### PR TITLE
Added support for OL9

### PR DESCRIPTION
**Pull request by feldsam**
Signed-off-by: Kristian Feldsam <feldsam@gmail.com>

### CODE REVIEW

File: tasks/repo-RedHat.yml [hunk 1]
```diff
@@ -14,11 +14,10 @@
       state: present
     when: ansible_distribution in [ 'RedHat' ]
 
-  # FIXME: this only works for 8
```
Comment (Habbie) [inline tasks/repo-RedHat.yml:17]: thanks!

**Comment by Habbie** [comment]
Thanks! Do you think you can add tests for ol9 too?

**Comment by feldsam** [comment]
Hi, I don't have experience with writting ansible tests

**Comment by Habbie** [comment]
It should be a matterof expanding `molecule/pdns-*/molecule.yml` with a few lines (for applicable versions).

**Comment by feldsam** [comment]
ok, I just added

**Comment by feldsam** [comment]
Hi, I just updated test suite to only include OL9 for 4.7 nad master

**Pull request merged**